sig-guidance.md

```diff
@@ -0,0 +1,23 @@
+# SIG Guidance
+
+A collection of guidance for Special Interest Groups.
+
+## Meeting Notes
+
+Notes must be taken during SIG meetings and made available to the public, which are linked from the SIG's README.md file in the governance repository.
```

Comment:
```suggestion
Notes must be taken during SIG meetings and made available to the public, linked from the SIG's README.md file in the governance repository.
```

To avoid the potential confusion that it would be "the public" that would be linked